### PR TITLE
HIGH PRIORITY: 0.13.1 hotfix release

### PR DESCRIPTION
**Pull request by dyladan**
https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/untagged-3e9c0596fe3f217993cf

Fixes https://github.com/open-telemetry/opentelemetry-js-contrib/issues/340

### CODE REVIEW

**Review by vmarchaud — APPROVED** [review]

**Comment by obecny** [comment]
@dyladan what is the fix here ? did you find the problem what causes that ?

**Comment by vmarchaud** [comment]
@obecny Missing "prepare" script: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/341/files#diff-4cb25a67d52949145eb6d4dd1b7ed4801a7e1125ab341911ca7fa69f0c7ec38bR19

**Review by obecny — APPROVED** [review]
lgtm

File: propagators/opentelemetry-propagator-ot-trace/src/version.ts [hunk 1]
```diff
@@ -15,4 +15,4 @@
  */
 
 // this is autogenerated file, see scripts/version-update.js
-export const VERSION = '0.13.0';
+export const VERSION = '0.13.1';
```
Comment (Flarna) [inline propagators/opentelemetry-propagator-ot-trace/src/version.ts:18]: Shouldn't we change the version.ts files of the others also?

Comment (dyladan) [reply propagators/opentelemetry-propagator-ot-trace/src/version.ts:18]: yes i was halfway through making this when a meeting started

Comment (dyladan) [reply propagators/opentelemetry-propagator-ot-trace/src/version.ts:18]: This afternoon I will bring over the tooling changes from the main repo which make this more automatic.

**Review by Flarna — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "opentelemetry-contrib",
-  "version": "0.13.0",
+  "version": "0.13.1",
   "description": "This is a repository for OpenTelemetry JavaScript contributions.",
-  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "repository": "https://github.com/open-telemetry/opentelemetry-js-contrib",
```
Comment (Flarna) [inline package.json:5]: I think this is not correct. if you want to use the full url you have to add also type and append .git
see https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

for github repos using just user/repo is fine.

Comment (dyladan) [reply package.json:5]: user/repo format broke the tool I use to generate the changelog

**Pull request merged**